### DISCUSSION TITLE
Perplexica just got faster!

### DISCUSSION THREAD

**Discussion by ItzCrazyKns** [ItzCrazyKns/Vane#401] [Announcements]
With the addition of optimization modes, Perplexica can generate answers in less than 3 seconds using the Groq provider. I'd love to hear everyone's thought about the quality of the new faster responses so I can work on the prompts and make them better.

**Comment by bergutman** [comment]
Works exceptionally well with Llama 3.2 11b!

**Comment by Chanduporalla** [comment]
The recent improvements to Perplexica with the addition of optimization modes and the use of the Groq provider sound promising, especially with the ability to generate responses in less than 3 seconds. It’s impressive to see a focus on boosting speed, but gathering feedback on the impact of these changes is a great step toward further refinement.

**Comment by ngdbao** [comment]
It's great, really.
But if api can expose something using streaming SSE, would be faster in first bytes
